Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -248,8 +248,7 @@ namespace internal {
             get(ecmap, e) ||
             get(fpm, face(h,pmesh))!=get(fpm, face(opposite(h,pmesh),pmesh)) )
       {
-        if (sqh < CGAL::squared_distance(get(vpmap, source(h, pmesh)),
-                                         get(vpmap, target(h, pmesh))))
+        if (sizing.is_too_long(source(h, pmesh), target(h, pmesh)))
```

Comment:
@janetournois I also updated the code here. Does it mean that it is incorrect? (in the 4/3 context).

Reply:
This is correct, the segment length will be compared to `4/3*target`

Reply:
After discussion we agreed that there should be another function in the sizing field `is_protected_constraint_too_long()` and it shoud also be used in the demo.

Reply:
@ipadjen @sloriot we should not forget that comment